modules/vaos/app/controllers/vaos/appointment_requests_controller.rb

```diff
@@ -4,21 +4,42 @@
 
 module VAOS
   class AppointmentRequestsController < ApplicationController
-    before_action :validate_params
+    before_action :validate_params, only: :index
+
+    MASSASSIGN_PARAMS_PUT = %i[
```

Comment:
Nitpick, but I'd prefer to see something like `MASSASSIGN_PARAMS_PUT` named something more like `MASS_ASSIGN_PUT_PARAMS`, for specific clarity

Reply:
At first glance I read "assassin params put"

Reply:
LOL I'll make this change this before merging.